### ISSUE TITLE
Notification: Send to -> Select a Field -> Should list Admin Email and not Email

### ISSUE THREAD

**Issue by paaljoachim** [fluentform/fluentform#462]
In Email Notifications: Send to - Select a Field
Send to Field: Email

![Screenshot 2024-08-05 at 17 10 31](https://github.com/user-attachments/assets/557374cc-9388-4109-9239-948dd633e9b4)

It should say Admin Email and not Email. It is confusing that it says Email which it should be Admin Email.

--

Further below under Advanced we have this:
![Screenshot 2024-08-05 at 17 11 09](https://github.com/user-attachments/assets/cb5ffa00-aa89-4045-be2e-b7fe1e266c42)
 
Option to add: Admin Email or Email.

**Comment by nkb-bd** [comment]
Admin notification email has admin email shortcode
<img width="847" alt="image" src="https://github.com/user-attachments/assets/1d423ea1-d5f4-4a5e-8654-1e865a2a9f7c">

**Comment by paaljoachim** [comment]
Yes. But the Admin shortcode does not have a 3 button link similar to the Subject line.
Also in the Advanced section below has a 3 button system.

![Screenshot 2024-09-12 at 08 27 15](https://github.com/user-attachments/assets/e6066e5b-cfcd-4e6f-843e-8b4825bf843e)
